muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/notes/NotesRESTService.java

```diff
@@ -211,7 +263,18 @@ public Response toggleArchived(@PathParam ("NOTEID") Long noteId) {
     }
 
     Note updatedNote = notesController.toggleArchived(note);
+    if(note.getArchived().equals(Boolean.FALSE)) {
+      note.setDueDate(null);
+      note.setStartDate(null);
```

Comment:
Both `dueDate` and `startDate` are reset in the entity but not propagated to the database?